algebird-core/src/main/scala/com/twitter/algebird/QTree.scala

```diff
@@ -258,6 +289,7 @@ case class QTree[A](
   }
 
   def interQuartileMean(implicit n: Numeric[A]): (Double, Double) = {
+    implicit val monoid: Monoid[A] = Ring.numericRing[A]
```

Comment:
Yep, since I think its far more likely the non-default monoid will be defined by the user in closer scope. But they'll pick up the default Numeric for the type. I doubt there'd be much issue with having the default Numeric and a custom Monoid. Currently here you'd start using the default monoid which would be bad.